libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/theme/ElementTheme.kt

```diff
@@ -66,6 +71,8 @@ fun ElementTheme(
         darkTheme -> materialDarkColors
         else -> materialLightColors
     }
+    var compoundColorPalette: CompoundColorPalette by remember { mutableStateOf(CompoundColorPalette.Light) }
+    compoundColorPalette = if (darkTheme) CompoundColorPalette.Dark else CompoundColorPalette.Light
```

Comment:
I had to add it like this to have the changes in light/dark mode affect these colours in real time, but maybe there's a better way to do it?

Reply:
Mmmmm, it shouldn't be necessary as long as we don't access those CompoundColorPalette values directly but only through the theme

Reply:
So we'd add them to `ElementColors` and remove the `LocalCompoundColors` object? I wanted to keep them separate until we have the Compound design tokens files with their final structure, so we can replace them easily. 

If we add them to `ElementColors` I think we'll need to re-work it so it picks different colors based on the `isLight` property.

Reply:
We should only have `elementColors` and `materialColors`
`elementColors` should already handle light/dark

Reply:
I just tried removing all usages of `LocalCompoundColors` and adding `quaternary` and `quinary`, but it seems like toggling dark mode doesn't update them:

https://user-images.githubusercontent.com/480955/221563871-cbed51d6-05e3-4302-a4d4-317bd9d7a90e.mov

Same goes for the `messageFrom*Background` colours:

<img width="593" alt="image" src="https://user-images.githubusercontent.com/480955/221564000-ca66629e-e599-40a7-a2f6-3ee244945ae0.png">